webanno-brat/src/main/java/de/tudarmstadt/ukp/clarin/webanno/brat/annotation/BratAnnotationEditor.html

```diff
@@ -26,6 +26,7 @@
         <div id="messagepullup" class="messages" style="display: none"></div>
         <div id="commentpopup"></div>
         <div id="document_name"></div>
+        <div id="span_btns_wrapper"></div>
```

Comment:
What if we display multiple brat annotator components on a single page? The reason why we have the IDs for the other brat legacy elements is - well - historic. Would we avoid having to assign a fixed ID this time?

Reply:
The reason we need ID is to remove/hide those buttons later when the mouse is out. However, I see your point here. I will see if we can use class instead.